src/main/drivers/exti.c

```diff
@@ -198,25 +199,25 @@ void EXTI_IRQHandler(void)
 }
 
 #define _EXTI_IRQ_HANDLER(name)                 \
-    void name(void) {                           \
+    IRQHANDLER(name) {                           \
         EXTI_IRQHandler();                      \
     }                                           \
     struct dummy                                \
     /**/
 
 
-_EXTI_IRQ_HANDLER(EXTI0_IRQHandler);
-_EXTI_IRQ_HANDLER(EXTI1_IRQHandler);
-#if defined(STM32F1) || defined(STM32F7)
-_EXTI_IRQ_HANDLER(EXTI2_IRQHandler);
-#elif defined(STM32F3) || defined(STM32F4)
-_EXTI_IRQ_HANDLER(EXTI2_TS_IRQHandler);
+_EXTI_IRQ_HANDLER(EXTI0_IRQ);
+_EXTI_IRQ_HANDLER(EXTI1_IRQ);
+#if defined(STM32F1) || defined(STM32F4) || defined(STM32F7)
```

Comment:
It is probably OK to special-case EXTI2_TS_IRQ and use EXTI2_IRQ as default for all other cases - IRQHANDLER will catch the error if default is not defined on new target